zaplings/templates/zaplings/creatorsnight.html

```diff
@@ -263,11 +258,17 @@ <h4 class="add-margin-bottom add-margin-top centerText">I want to meet someone w
             $('#selected-needs').on('click', 'label', (function(){
                 $(this).toggleClass('selected').blur();
             }));
-        });
         
-        $('#open-mic').click(function(){
-            $('#open-mic-text').toggleClass('hidden');
+            $('#open-mic').click(function(){
+                $('#open-mic-text').toggleClass('hidden');
+            });
+
+            $('#form-rsvp').submit(function() {
```

Comment:
@dannypernik I changed this part - I think it is the form itself that needs to be serialized